script/deploy/tron/register-facets-to-diamond.ts

```diff
@@ -0,0 +1,750 @@
+#!/usr/bin/env bun
+
+import * as fs from 'fs'
+import * as path from 'path'
+
+import { defineCommand, runMain } from 'citty'
+import { consola } from 'consola'
+import { TronWeb } from 'tronweb'
+
+import { EnvironmentEnum, type SupportedChain } from '../../common/types'
+import { getPrivateKeyForEnvironment } from '../../demoScripts/utils/demoScriptHelpers'
+
+import {
+  getEnvironment,
+  ENERGY_PRICE,
+  updateDiamondJsonBatch,
+} from './utils.js'
+
+// Verified selectors (from contract-selectors.sh)
+const FACET_SELECTORS: Record<string, string[]> = {
```

Comment:
Consider extracting them from the artifact in the out folder, the function selectors are all there for each contract

Also, please document (somewhere, e.g. notion) how a new facet can be added and which files need to be created/updated for this

Reply:
Please share the notion page in our Slack channel once created. 
You may resolve this issue here.